wren-ai-service/src/providers/document_store/qdrant.py

```diff
@@ -265,7 +265,7 @@ async def write_documents(
 
         document_objects = self._handle_duplicate_documents(
             documents=documents,
-            index=self.index,
```

Comment:
im curious why we comment this out.

Reply:
new version of qdrant sdk doesn't need that anymore